addons/api/addon/models/auth-method.js

```diff
@@ -1,9 +1,31 @@
 import GeneratedAuthMethodModel from '../generated/models/auth-method';
+import { fragment } from 'ember-data-model-fragments/attributes';
 import { equal } from '@ember/object/computed';
 
+export const signingAlgorithms = [
```

Comment:
This isn't being used anywhere. Is this for reference in future PRs?

Reply:
Yes.  We don't necessarily have a great way to model `enum`-type fields.  But UI code will definitely need to know about allowable values to display them in, say, a `select`.